### PR TITLE
[SCIP, SCIP_PAPILO] Update to 8.0.4

### PR DESCRIPTION
**Pull request by claud10cv**
Updated build_tarballs to build version 8.0.4

### CODE REVIEW

**Comment by matbesancon** [comment]
thanks @claud10cv !

**Review by matbesancon — APPROVED** [review]

File: S/SCIP/build_tarballs.jl [hunk 1]
```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "SCIP"
-version = v"800.0.301"
+version = v"800.0.401"
```
Comment (matbesancon) [inline S/SCIP/build_tarballs.jl:6]: I think we could have 800.0.400 here?

**Review by matbesancon — APPROVED** [review]

**Comment by matbesancon** [comment]
maybe SCIP_jll will even work on windows now :eyes:

**Comment by matbesancon** [comment]
actually @claud10cv did you try with the bliss patch removed? It should work now

**Comment by claud10cv** [comment]
How Do I do That? Is it as simply as removing the file?

**Comment by matbesancon** [comment]
and the atomic patch: https://github.com/JuliaPackaging/Yggdrasil/pull/7308/files#diff-9830d59b5ee404d92a00fd4e0cf34acd9c2bfa76346412df26cd1f0d92892ed5R28

File: S/SCIP/build_tarballs.jl [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
 
 name = "SCIP"
-version = v"800.0.301"
+version = v"800.0.400"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://scipopt.org/download/release/scipoptsuite-8.0.3.tgz", "5ad50eb42254c825d96f5747d8f3568dcbff0284dfbd1a727910c5a7c2899091"),
+    ArchiveSource("https://scipopt.org/download/release/scipoptsuite-8.0.4.tgz", "be4f978be7f8f97371ddcdac7a60af69a4fea5f975090fe35f1ae4308db692d3"),
     DirectorySource("./bundled"),
```
[the remark this answers is not in the document]
Comment (claud10cv) [reply S/SCIP/build_tarballs.jl:11]: for scip_papilo as well? (I will apply the changes)

[the remark this answers is not in the document]
Comment (matbesancon) [reply S/SCIP/build_tarballs.jl:11]: yes it should work for both

**Comment by matbesancon** [comment]
wonderful, all good now!

**Review by matbesancon — APPROVED** [review]

**Pull request merged**